scipy/signal/_max_len_seq.py

```diff
@@ -72,8 +72,9 @@ def max_len_seq(nbits, state=None, length=None, taps=None):
 
     MLS has a white spectrum (except for DC):
 
+    >>> import numpy as np
     >>> import matplotlib.pyplot as plt
-    >>> from numpy.fft import fft, ifft, fftshift, fftfreq
+    >>> from np.fft import fft, ifft, fftshift, fftfreq
```

Comment:
This doesn't work--a variable name can't be used in an import statement like this.
```suggestion
    >>> from numpy.fft import fft, ifft, fftshift, fftfreq
```